docs/general/serializers.md

```diff
@@ -74,6 +74,30 @@ def blog
 end
 ```
 
+### Polymorphic Relationships
+
+Polymorphic relationships are serialized just by specifying the relationship. For example:
```

Comment:
Ie like any other relationships since the object itself resolves the associated object, which ams treats no sifferently from any other object